core/include/irods/private/http_api/common.hpp

```diff
@@ -169,6 +169,8 @@ namespace irods::http
 
 	auto map_json_to_user(const nlohmann::json& _json) -> std::optional<std::string>;
 
+	auto validate_using_local_validation(std::string _thing) -> std::optional<nlohmann::json>;
```

Comment:
perhaps... `_token` or `_jwt` instead of `_thing`?

Reply:
Removed, not used outside of `common.cpp`.